code/game/objects/items/cigs_lighters.dm

```diff
@@ -132,6 +132,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	var/chem_volume = 30
 	var/smoke_all = FALSE /// Should we smoke all of the chems in the cig before it runs out. Splits each puff to take a portion of the overall chems so by the end you'll always have consumed all of the chems inside.
 	var/list/list_reagents = list(/datum/reagent/drug/nicotine = 15)
+	var/lung_harm = 1.2 //How bad it is for you

```

Comment:
Some math: First, remember that cigs tick 1/5 as slow as the main life tick, so you'd want it to be .55. Second, since the damage per tick is x-.5, net lung damage per minute is six times that, and net lung damage per cig is six times that. At .55 per tick, that's .3 lung damage per minute - you'd need upward of five hours for it to do anything. At 1/, it's 18 per cig - just north of 30 minutes for lungs to fail. At 1.2, it's about 25, and 25 to fail. As far as I can tell, lung damage doesn't actually do anything before organ failure besides making you breath faster, which is pretty insignificant. Incidentally, 1/ is the point at which 6 minutes smoking/6 not smoking is lung neutral.
Cigars run out after 10-30 minutes, depending on brand. Determined by smoketime var, which gives process tick number before expiration.